### ISSUE TITLE
[Feature]: Sort Tailwind CSS classes in HTML attributes other than "class"

### ISSUE THREAD

**Issue by kubikill** [shufo/blade-formatter#797]
### Description

It would be great if there was an option to have blade-formatter also sort Tailwind CSS classes in HTML attributes other than `class`.
This would be useful for Blade components that use HTML attributes other than `class` to add classes to child elements of the component. 
Any chance this could be added to prettier-plugin-blade as well?

### Suggested Solution

Add an option that would allow to specify in which HTML attributes blade-formatter should sort Tailwind CSS classes.

**Comment by beholdr** [comment]
Yeah, this would be great! Because it's a common pattern to set components classes like this `{{ $attributes->class('border border-gray-200 rounded-lg overflow-hidden divide-y divide-gray-200') }}` and currently it's not get sorted.

**Comment by beholdr** [comment]
@shufo is this feature really not planned?
